src/action.ts

```diff
@@ -19,27 +19,46 @@ import { InputOptions } from '@actions/core'
 import * as fs from 'fs'
 import * as gpg from './gpg'
 import * as sops from './sops'
+import * as envfile from 'envfile'
+import * as yaml from 'js-yaml'
 
-export async function run() {
+export async function run () {
   try {
     const required: InputOptions = {
       required: true
     }
     const version: string = core.getInput('version', required)
-    const gpg_key: string = core.getInput('gpg_key', required)
-    const encrypted_file: string = core.getInput('file', required)
-    const output_type: string = core.getInput('output_type')
-    let outputFormat = await sops.getOutputFormat(output_type)
-    let sopsPath = await sops.install(version, fs.chmodSync)
-    await gpg.import_key(gpg_key)
-    let result: string = await sops.decrypt(sopsPath, encrypted_file, outputFormat)
+    const gpgKey: string = core.getInput('gpg_key', required)
+    const encryptedFile: string = core.getInput('file', required)
+    const outputType: string = core.getInput('output_type')
+    const outputFormat = await sops.getOutputFormat(outputType)
+    const sopsPath = await sops.install(version, fs.chmodSync)
+    await gpg.importKey(gpgKey)
+    let result: string = await sops.decrypt(sopsPath, encryptedFile, outputFormat)
+
+    hideSecrets(result, outputFormat)
```

Comment:
In my view we should add tests to see if secrets are masked.

Reply:
We are just calling here standard API function, not really sure how do you want to test it (esp. that you would need a test run on GitHub)

Reply:
Yes, you are correct. 😄 
I think we can add some test which makes sure that the standard API function is triggered so that this test fails if the masking is removed at later point in time.

Reply:
@navyatejaotturu-metronom Good tip, journey tests are updated